tests/ext/extract_ip_addr.phpt

```diff
@@ -18,9 +18,11 @@ function test($header, $value) {
 test('x_forwarded_for', '2001::1');
 test('x_forwarded_for', '::1, febf::1, fc00::1, fd00::1,2001:0000::1');
 test('x_forwarded_for', '172.16.0.1');
-test('x_forwarded_for', '172.16.0.1, 172.31.255.254, 172.32.255.1, 8.8.8.8');
+test('x_forwarded_for', '172.16.0.1, 172.31.255.254, 172.32.255.1, 8.8.8.8, 1.2.3.4:456, [2001::1]:1111');
 test('x_forwarded_for', '169.254.0.1, 127.1.1.1, 10.255.255.254,');
 test('x_forwarded_for', '127.1.1.1,, ');
+test('x_forwarded_for', '1.2.3.4:456');
+test('x_forwarded_for', '[2001::1]:1111');
```

Comment:
Can you add to this test an `x_forwarded`, `via` and `forwarded` variant using IPv6 with no port as a regression? For example (but you can add a few more with lists and such):

```
test('x_forwarded', 'for="2001:abcf::1"');
test('forwarded', 'for=2001:abcf:1f::55');
test('via', '2001:abcf:1f::55');

Reply:
Done